classes/Employee.php

```diff
@@ -423,12 +453,19 @@ public function isLastAdmin()
      */
     public function setWsPasswd($passwd)
     {
+        try {
```

Comment:
Same question, we don't expect the object construction to fail here, more you shouldn't ignore the exception :)